### ISSUE TITLE
Error when I try and run it

### ISSUE THREAD

**Issue by waz86** [FabricMC/fabric-example-mod#108]
I just got the example mod set up, but whenever I try and run it I get this error:

Exception in thread "main" java.lang.RuntimeException: Failed to read accessWidener file from mod fabric-object-builder-api-v1
	at net.fabricmc.loader.FabricLoader.loadAccessWideners(FabricLoader.java:441)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:129)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at net.fabricmc.devlaunchinjector.Main.main(Main.java:86)
Caused by: java.lang.RuntimeException: Namespace (intermediary) does not match current runtime namespace (named)
	at net.fabricmc.accesswidener.AccessWidenerReader.read(AccessWidenerReader.java:47)
	at net.fabricmc.loader.FabricLoader.loadAccessWideners(FabricLoader.java:439)
	... 3 more

I've looked all over but can't find a solution, please help if you can, thank you!

**Comment by sfPlayer1** [comment]
This looks like an intermediary (=end user distribution) Fabric API being used in-dev, presumably through having put it in the mods folder?

**Comment by waz86** [comment]
> This looks like an intermediary (=end user distribution) Fabric API being used in-dev, presumably through having put it in the mods folder?

Sorry I didn't respond sooner, can you dumb it down some, I'm pretty new to this and don't know what you mean lol

**Comment by wyvest** [comment]
> > This looks like an intermediary (=end user distribution) Fabric API being used in-dev, presumably through having put it in the mods folder?
> 
> Sorry I didn't respond sooner, can you dumb it down some, I'm pretty new to this and don't know what you mean lol

what I'm assuming it means is that you have fabric API in your dev env mod folder (you're not supposed to do that)

**Comment by waz86** [comment]
> > > This looks like an intermediary (=end user distribution) Fabric API being used in-dev, presumably through having put it in the mods folder?
> 
> > 
> 
> > Sorry I didn't respond sooner, can you dumb it down some, I'm pretty new to this and don't know what you mean lol
> 
> 
> 
> what I'm assuming it means is that you have fabric API in your dev env mod folder (you're not supposed to do that)

Ah ok lol thanks I'll try that
